packages/kbn-custom-icons/src/components/agent_icon/get_agent_icon.ts

```diff
@@ -66,6 +66,14 @@ const darkAgentIcons: { [key: string]: string } = {
   rust: darkRustIcon,
 };
 
+const sanitizeAgentName = (agentName: string) => {
+  if (agentName.startsWith('opentelemetry/')) {
```

Comment:
I think we also need this for `otlp` - this is the agent name in our PoC env using the otel collector for reference (we can also change the scenario to cover both)

Reply:
solved 👍